cfgov/jinja2/v1/owning-a-home/loan-estimate/index.html

```diff
@@ -38,48 +38,53 @@
+                                    "email_text": "Check the details on your mortgage Loan Estimate and get definitions with this interactive tool from the CFPB.",
+                                    "email_signature": "-- From the CFPB",
+                                    "linkedin_title": "Learn the lingo of Loan Estimates with the @CFPB.",
+                                    "linkedin_text": "The @CFPB has a helpful tool to review your mortgage Loan Estimate and get definitions for unfamiliar terms. Via @CFPB"
+                                } ) }}
+                            </div>
+                        </div>
+                        <div class="content-l_col-1-3">
+                            <img src="{{ static('apps/owning-a-home/img/ill-form-explainer-loan-est.png') }}"
+                                alt=""
+                                class="illustration">
+                        </div>
                     </div>
-                </div>
-                <div class="content-l_col-1-3">
-                    <img src="{{ static('apps/owning-a-home/img/ill-form-explainer-loan-est.png') }}"
-                         alt=""
-                         class="illustration">
-                </div>
-            </div>
-            {# Import the form explainer macros #}
-            {% import "_templates/form-explainer.html" as form_explainer %}
-            {# Import the data to use with form explainer #}
-            {% from "loan-estimate-1-data.html" import data as page_1 %}
-            {% from "loan-estimate-2-data.html" import data as page_2 %}
-            {% from "loan-estimate-3-data.html" import data as page_3 %}
-            {# Render form explainer with the data for this form #}
-            <div class="col-12">
-                {{ form_explainer.render([ page_1, page_2, page_3 ]) }}
-            </div>
-            <div class="col-12">
-                {% include "_templates/ratings-form.html" %}
+                    {# Import the form explainer macros #}
+                    {% import "_templates/form-explainer.html" as form_explainer %}
+                    {# Import the data to use with form explainer #}
+                    {% from "loan-estimate-1-data.html" import data as page_1 %}
+                    {% from "loan-estimate-2-data.html" import data as page_2 %}
+                    {% from "loan-estimate-3-data.html" import data as page_3 %}
+                    {# Render form explainer with the data for this form #}
+                    <div class="col-12">
+                        {{ form_explainer.render([ page_1, page_2, page_3 ]) }}
+                    </div>
+                    <div class="col-12">
+                        {% include "_templates/ratings-form.html" %}
+                    </div>
+                </div><!-- /.form-explainer -->
             </div>
-        </div><!-- /.form-explainer -->
+        </div>
```

Comment:
I think this is an unnecessary edition related to the extra indentation above.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4782/commits/2c86e63a4ef4b0dbb782442355027186875274a2